content/en/tracing/service_catalog/setup.md

```diff
@@ -36,6 +39,24 @@ If the service, API, or custom library you are interested in is not listed on th
 
 To discover other services through existing Datadog telemetry such as infrastructure metrics, go to the **Discover Services** tab and follow instructions there. Discovery uses the `DD_SERVICE` [Unified Service Tagging convention][4] to locate services sending data to your Datadog organization.
 
+## Set up the GitHub Apps integration
+
+You can use the [GitHub Apps integration][5] to display links for service definitions in GitHub.
+
+To install the GitHub Apps integration, navigate to the [integration tile][6] and click **Link GitHub Account** in the **Repo Configuration** tab. 
+
+### Service definition YAML files
+
+Datadog scans for the `service.datadog.yaml` file at the root of each repository with read permissions. You can register multiple services in one YAML file by creating multiple YAML documents. Separate each document with three dashes (`---`).
+
+### Modify service definition
+
+The **Edit in Github** button appears in the **Definition** tab and links users directly to a pre-templated commit that you can modify.
+
+{{< img src="tracing/service_catalog/svc_cat_contextual_link.png" alt="An Edit in Github button appears in the Definition tab of a service in the Service Catalog" style="width:90%;" >}}
```

Comment:
this screen cap shows too much UI to easily locate the button described. Maybe crop a bit or circle the button (or both).